boubleSort/src/main/java/sci/java/SorterByAge.java

```diff
@@ -0,0 +1,7 @@
+package sci.java;
+
+public interface SorterByAge {
```

Comment:
The method names on the interface you've chosen have no relation to sorting.

A more correct approach would be
```
public interface IntSortable {
  void getIntCriteriaForSorting(); 
}
```
once we learn generics the interface would turn into
```
public interface Sortable<T> {
    T getSortingCriteriaValue();
}

public class Person implements Sortable<Integer> {

    private int age;

    // rest of the code

    @Override
    public Integer getSortingCriteriaValue() {
        return age;
    }
}

```
Also, once we learn collections we'll see how the Comparable and Comparator interfaces help.